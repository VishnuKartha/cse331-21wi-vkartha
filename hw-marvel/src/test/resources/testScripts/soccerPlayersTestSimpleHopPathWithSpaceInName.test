# Tests that the simple hop between two players in the soccer data base works

# To run this test, the soccer dataset must exist at
#   hw-marvel/src/main/resources/data/soccer.tsv

LoadGraph soccer soccer.tsv

FindPath soccer Leo_Messi Mbappe
FindPath soccer Mbappe Leo_Messi

