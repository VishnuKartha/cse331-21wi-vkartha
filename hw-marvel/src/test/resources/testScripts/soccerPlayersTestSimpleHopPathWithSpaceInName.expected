# Tests that the simple hop between two players in the soccer data base works

# To run this test, the soccer dataset must exist at
# hw-marvel/src/main/resources/data/soccer.tsv

loaded graph soccer

path from Leo Messi to Mbappe:
Leo Messi to Neymar via Barcelona
Neymar to Mbappe via PSG
path from Mbappe to Leo Messi:
Mbappe to Neymar via PSG
Neymar to Leo Messi via Barcelona